apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -15,13 +16,47 @@ class ManageStudentsLoginInfo extends Component {
     sectionId: PropTypes.number,
     sectionCode: PropTypes.string,
     loginType: PropTypes.string,
-    // The prefix for the code studio url in the current environment,
-    // e.g. '//studio.code.org' or '//localhost-studio.code.org:3000'.
-    studioUrlPrefix: PropTypes.string,
+  };
+
+  prependUrlOrigin = (url) => {
+    if (url.startsWith('//')) {
+      return `http:${url}`;
+    } else {
+      return window.location.origin + url;
+    }
+  };
+
+  joinSectionUrl = (loginType) => {
+    let path = '';
+
+    if (loginType === SectionLoginType.word || loginType === SectionLoginType.picture) {
+      path = pegasus('/join');
+    } else if (loginType === SectionLoginType.email) {
+      path = studio(`/join/${this.props.sectionCode}`);
+    }
+
+    // This URL is displayed as a raw URL to the user, so it needs to include origin (not just a relative path)
+    // if it doesn't already.
+    return this.prependUrlOrigin(path);
+  };
+
+  sectionSignInUrl = () => {
+    const path = studio(`/sections/${this.props.sectionCode}`);
+    // This URL is displayed as a raw URL to the user, so it needs to include origin (not just a relative path)
+    // if it doesn't already.
+    return this.prependUrlOrigin(path);
+  };
+
+  printSignInCardsUrl = () => {
+    return pegasus(`/teacher-dashboard#/sections/${this.props.sectionId}/print_signin_cards`);
```

Comment:
This url is tricky because it points back to the old teacher dashboard. are you planning to move the print_signin_cards functionality over to dashboard? I don't see any UI tests for the print_signin_cards route on pegasus, so you'll have to remember not to break this one when you go to shut down teacher-dashboard on pegasus.